nextflow.config

```diff
@@ -130,6 +130,11 @@ profiles {
         docker.enabled = true
     }
 
+    conda {
+        conda.enabled = true
+        process.conda = 'environment-pipeline.yml'
+    }
+
```

Comment:
Nice! Should we add a second example command to the README?

At some point the README could be cleaned up to more clearly provide what setup is necessary for either runtime environment (e.g. the Docker images aren't needed for this profile).

https://github.com/seqwell/LongPlex?tab=readme-ov-file#example-command